CHANGELOG.md

```diff
@@ -4,7 +4,9 @@
 
 ##### Enhancements
 
-* None.  
+* Add User-Agent to `cURL` requests when downloading source via the `:http` download strategy, unless one was provided by the `:headers` option.
```

Comment:
nit, please add two empty spaces at the end of this line so markdown formatting works.